instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -696,19 +696,28 @@ async def otel_send(message: dict[str, Any]):
             ) as send_span:
                 if callable(self.client_response_hook):
                     self.client_response_hook(send_span, scope, message)
-                if send_span.is_recording():
-                    if message["type"] == "http.response.start":
-                        status_code = message["status"]
+
+                status_code = None
+                if message["type"] in {
+                    "http.response.start",
+                    "websocket.send",
+                }:
+                    status_code = (
+                        message["status"]
+                        if message["type"] == "http.response.start"
+                        else 200
+                    )
+                    if status_code:
                         duration_attrs[SpanAttributes.HTTP_STATUS_CODE] = (
                             status_code
                         )
```

Comment:
```suggestion
                if message["type"] == "http.response.start":
                    status_code = message["status"]
                    if status_code:
                        duration_attrs[SpanAttributes.HTTP_STATUS_CODE] = (
                            status_code
                        )
                elif message["type"] == "websocket.send":
                    status_code = 200
                    duration_attrs[SpanAttributes.HTTP_STATUS_CODE] = (
                        status_code
                    )
```

What do you think on something like this instead?

Reply:
Had something like that originally but ran into `R0912: Too many branches (13/12) (too-many-branches)`.

Went back to a clearer if/else and did a slight refactor to pull out some logic to avoid the lint rule.